app/assets/javascripts/vulnerabilities.js.coffee

```diff
@@ -104,7 +104,7 @@ filterReleasesByMajorVersion = (releases, majorVersion) ->
       ///^#{majorVersion}///.test(release.version)
   else
     releases.filter (release) ->
-      ///^#{majorVersion}+(?:\.\d+)+$///.test(release.version)
+      ///^(v)+#{majorVersion}+(?:\.\d+)+$///.test(release.version)
```

Comment:
Change `(v)+` to `v?`. 
`?` means `v` is optional and `()` is used for optional matches or when we are going to use the value after matching.